src/plugins/walletConnectToDapps/utils/EIP155RequestHandlerUtil.ts

```diff
@@ -90,14 +90,19 @@ export const approveEIP155Request = async ({
       const fees = await getFeesForTx(sendTransaction, chainAdapter, accountId)
       const senderAddress = await chainAdapter.getAddress({ accountNumber, wallet })
       const gasData = getGasData(customTransactionData, fees)
+      const gasLimit = (() => {
+        if (customTransactionData.gasLimit) return customTransactionData.gasLimit
+        if (sendTransaction.gasLimit) return sendTransaction.gasLimit
+        return '90000' // https://docs.walletconnect.com/2.0/advanced/rpc-reference/ethereum-rpc#eth_sendtransaction
```

Comment:
gas limit is optional, defaulting to 90000 on WC side, so maybe we can return undefined here and rely on WC default on their side?

Reply:
Oh i see that `gasLimit` is non-optional on our side, ignore me